src/lib/intent/entry-transform.ts

```diff
@@ -8,12 +8,11 @@ export default class EntryTransformIntent extends Intent {
     return true
   }
 
-  groupsWith (other: Intent): boolean {
-    const sameContentType = other.getContentTypeId() === this.getContentTypeId()
-    return other.isContentTransform() && sameContentType
```

Comment:
So we decided to never group content transforms together? What changed our mind?

Reply:
I think at the time we weren't 100% sure if that could have any negative effects and decided to err on the side of caution.

Reply:
That we would have to explicitly state how to handle the case where one content transform publishes the entries but the other doesn't.